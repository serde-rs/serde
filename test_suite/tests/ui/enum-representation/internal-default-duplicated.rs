use serde_derive::Deserialize;

#[derive(Deserialize)]
#[serde(tag = "tag")]
enum E {
    #[serde(default)]
    V1 { f: u8 },
    #[serde(default)]
    V2 { f: u8 },
}

fn main() {}
