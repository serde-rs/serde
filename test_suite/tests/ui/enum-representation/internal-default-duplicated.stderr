error: #[serde(default)] already defined at variant V1
 --> tests/ui/enum-representation/internal-default-duplicated.rs:8:5
  |
8 | /     #[serde(default)]
9 | |     V2 { f: u8 },
  | |________________^
